programmers/난이도별/level01.나누어_떨어지는_숫자_배열/rockmiin.py

```diff
@@ -0,0 +1,13 @@
+def solution(arr, divisor):
+    answer=[]
+    for i in arr:
+        if i % divisor ==0:
+            answer.append(i)
+    if(answer==[]): answer.append(-1)
+    return sorted(answer)
```

Comment:
```suggestion
    return [-1] if answer == [] else sorted(answer)
```
간단한 경우의 수는 이렇게 한 줄로 줄일수 있습니다!